UI/window-basic-filters.cpp

```diff
@@ -179,6 +187,15 @@ void OBSBasicFilters::UpdatePropertiesView(int row, bool async)
 			OBSBasicFilters::UpdateProperties,
 			this);
 
+	uint32_t caps = obs_source_get_output_flags(filter);
+	if (drawablePreview && (caps & OBS_OUTPUT_VIDEO)) {
```

Comment:
This should probably be `OBS_SOURCE_VIDEO` instead of `OBS_OUTPUT_VIDEO`, even if their actual values are the same for the time being